V3.0/UFTP-V3.0-metering.xsd

```diff
@@ -0,0 +1,157 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema" elementFormDefault="qualified" attributeFormDefault="unqualified">
+
+    <xs:include schemaLocation="UFTP-V3.0-common.xsd"/>
+    <xs:include schemaLocation="UFTP-V3.0-agr-dso.xsd"/>
+
+    <!-- -->
+    <!-- EAN Type -->
+    <!-- -->
+    <xs:simpleType name="EANType">
+        <xs:restriction base="xs:string">
+            <xs:pattern value="[Ee][0-9]{16}"/>
+        </xs:restriction>
+    </xs:simpleType>
+    <!-- -->
+    <!-- -->
+    <!--  Metering Unit Type -->
+    <!-- -->
+    <xs:simpleType name="MeteringUnitType">
+        <xs:restriction base="xs:string">
+            <xs:enumeration value="kW">
+                <xs:annotation>
+                    <xs:documentation>kW must be used with Power profile values.</xs:documentation>
+                </xs:annotation>
+            </xs:enumeration>
+            <xs:enumeration value="kWh">
+                <xs:annotation>
+                    <xs:documentation>kWh must be used with energy profile values (ImportEnergy,ExportEnergy,ImportMeterReading,ExportMeterReading).</xs:documentation>
+                </xs:annotation>
+            </xs:enumeration>
+        </xs:restriction>
+    </xs:simpleType>
+    <!-- -->
+    <!--  Metering Profile Enum -->
+    <!-- -->
+    <xs:simpleType name="MeteringProfileEnum">
+        <xs:restriction base="xs:string">
+            <xs:enumeration value="Power">
+                <xs:annotation>
+                    <xs:documentation>The average active power during ISP, considering both import and export energy. Power=(ImportEnergy-ExportEnergy)*2
```

Comment:
Is this description correct? If the ISP is 15 minutes, and ImportEnergy-ExportEnergy = 100kWh, this leads to 200kW but it's actually 800kW (if 200kWh is delivered over 1/4th of an hour), but if the ISP is 1 hour, power would be the same in this equation (=200kW), but it's actually only 1/4th because the same amount of energy is released over a longer period of time. Let me know if I misinterpreted this!

Reply:
You are correct.  This equation assumes a half-hour duration for the ISP.

I will correct this to: Power=(ImportEnergy-ExportEnergy)*(60/ISPDurationMinutes)